scripts/idl/matter_grammar.lark

```diff
@@ -63,23 +71,21 @@ struct_field: member_attribute* field
 member_attribute: "optional"i -> optional
                 | "nullable"i -> nullable
 
-field: data_type id list_marker? "=" number ";"
+field: data_type id list_marker? "=" number
 list_marker: "[" "]"
 
 data_type: type ("<" number ">")?
 
 id: ID
 type: ID
 
-COMMENT: "{" /(.|\n)+/ "}"
-       | "//" /.*/
```

Comment:
Provide a textual example of candidates of how this looks just above

Reply:
This was a removal of code. What would you like me to change here?

Generally I tried to put examples in the readme.md to not duplicate them both in lark and readme (I believe readme should have examples). Can add examples here as well, however I am not convinced things would be readable.